streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1847,8 +1847,7 @@ void maybeInitTaskTimeoutsOrThrow(
         final long nowMs
     ) {
         for (final TopicPartition partition : partitions) {
-            final Task task = getActiveTask(partition);
-            task.maybeInitTaskTimeoutOrThrow(nowMs, timeoutException);
```

Comment:
ty for boiling down.

Now we have reached final version, can not take away anymore.